bin/oneoff/nces_data/import_school_stats_by_years_2014_2015_ccd

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+CDO.log = Logger.new(STDOUT)
+
+AWS::S3.process_file('cdo-nces', '2014-2015/ccd/ccd_sch_029_1415_w_0216601a.txt') do |filename|
+  SchoolStatsByYear.transaction do
+    SchoolStatsByYear.merge_from_csv(filename) do |row|
+      {
+        school_id:          row['NCESSCH'].to_i.to_s,
+        school_year:        row['SURVYEAR'].presence || '2014-2015',
+        grades_offered_lo:  row['GSLO'],
+        grades_offered_hi:  row['GSHI'],
+        grade_pk_offered:   row['PKOFFERED'][0, 1] == 'Y',
+        grade_kg_offered:   row['KGOFFERED'][0, 1] == 'Y',
+        grade_01_offered:   row['G1OFFERED'][0, 1] == 'Y',
+        grade_02_offered:   row['G2OFFERED'][0, 1] == 'Y',
+        grade_03_offered:   row['G3OFFERED'][0, 1] == 'Y',
+        grade_04_offered:   row['G4OFFERED'][0, 1] == 'Y',
+        grade_05_offered:   row['G5OFFERED'][0, 1] == 'Y',
+        grade_06_offered:   row['G6OFFERED'][0, 1] == 'Y',
+        grade_07_offered:   row['G7OFFERED'][0, 1] == 'Y',
+        grade_08_offered:   row['G8OFFERED'][0, 1] == 'Y',
+        grade_09_offered:   row['G9OFFERED'][0, 1] == 'Y',
+        grade_10_offered:   row['G10OFFERED'][0, 1] == 'Y',
+        grade_11_offered:   row['G11OFFERED'][0, 1] == 'Y',
+        grade_12_offered:   row['G12OFFERED'][0, 1] == 'Y',
+        grade_13_offered:   row['G13OFFERED'][0, 1] == 'Y',
+        virtual_status:     row['VIRTUAL']
+      }
+    end
+  end
+end
+
+AWS::S3.process_file('cdo-nces', '2014-2015/ccd/ccd_sch_052_1415_w_0216161a.txt') do |filename|
+  SchoolStatsByYear.transaction do
+    SchoolStatsByYear.merge_from_csv(filename) do |row|
+      {
+        school_id:          row['NCESSCH'].to_i.to_s,
+        school_year:        row['SURVYEAR'].presence || '2014-2015',
+        students_total:     row['TOTAL'].presence.try {|v| v.to_i <= 0 ? nil : v.to_i},
```

Comment:
`presence.try {|v| v.to_i <= 0 ? nil : v.to_i}` is duplicated a lot and could use refactoring into a helper method for readability